src/powershell/install.sh

```diff
@@ -162,6 +163,12 @@ if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
     done
 fi
 
+# If URL for powershell profile is provided, download it to '/opt/microsoft/powershell/7/profile.ps1'
+if [ -n "$POWERSHELL_PROFILE_URL" ]; then
```

Comment:
I like the idea of being able to configure the Powershell profile with the Powershell Feature. Can we convert this to a [Feature option](https://containers.dev/implementors/features/#options-property)?

I'd expect it to be a `string` and called something like `powershellProfileURL` which is an optional param set to empty string.

Reply:
Done :)